backend/src/services/org/org-service.ts

```diff
@@ -118,6 +132,54 @@ export const orgServiceFactory = ({
     return workspaces.filter((workspace) => organizationWorkspaceIds.has(workspace.id));
   };
 
+  const addGhostUser = async (orgId: string, tx?: Knex) => {
+    const email = `ghost@${nanoid(16)}-${orgId}.com`; // We add a nanoid because the email is unique. And we have to create a new ghost user each time, so we can have access to the private key.
```

Comment:
> Use the alphaNum nanoId in lib. The above will generate symbols which would practically make the email invalid.

The only type of special characters that `nanoid` uses, is `_` and `-`, which is fine.

> This would ensure if there is any point of email verification being done its still a valid email. Another thing use @infisical because that would reduce scope of a domain attack.

Email verification won't be done, as it's not possible to log in as the ghost user at any point. However I've added `@infisical`.

Reply:
I am not talking about the login based email verification. Generally speaking if we run some migration and there existed an email validation checks its better the email above to valid ones rather than invalid emails